dev/update_requirements.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2020 The Flax Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility script for updating examples' "requirements.txt"
+
+WARNING: Updates examples/*/requirements.txt in place, so you probably want to
+run this in a clean working directory...
+"""
+
+import glob
+import pathlib
+import re
+import sys
+
+
+import_re = re.compile(r'(?:from|import)\s+(\w+)')
+pkg_map = {
+  'absl': 'absl-py',
+  'atari_py': 'atari-py',
+  'cv2': 'opencv-python',
+  'ml_collections': 'ml-collections',
+  'PIL': 'Pillow',
+  'tensorflow_datasets': 'tensorflow-datasets',
+  'tensorflow_text': 'tensorflow-text',
+}
+standard_libs = set('codecs collections dataclasses datetime enum functools math multiprocessing itertools os pathlib random re sys tempfile time typing unicodedata warnings'.split(' '))
+
+
+if len(sys.argv) != 2:
+  print()
+  print('Expected single argument that has form "clu-0.0.6 flax-0.3.6 ignoreme- ..."')
```

Comment:
Can you explain what the meaning of the single argument is? I guess it will override this version in requirements.txt of all examples?

Reply:
You could also decide to use `absl.flags` and explain it in the flag...